scripts/apps/monitoring/services/CardsService.ts

```diff
@@ -76,33 +89,40 @@ export function CardsService(search, session, desks, $location) {
 
     function filterQueryByCardType(query, queryParam, card: ICard) {
         let deskId;
+        const extensionSection = getExtensionSections();
+
+        extensionSection.forEach((response) => {
+            if (response.id === card.type) {
+                query.filter(response.query);
```

Comment:
`query.filter` should never be called multiple times. It wouldn't be, because you are checking the ID, but it is still confusing that it is called inside a loop. Instead use `find` to get a single item, check if it's found and only call then.